src/foam/lib/csv/CSVOutputterImpl.js

```diff
@@ -114,7 +115,10 @@ foam.CLASS({
           getSb().append(value);
         } else if ( value instanceof Date ) {
           if ( getSheetsCompatible() ) {
-            getSb().append("Sheets: " + value); // TODO: use real format
+            Date date = (Date) value;
+            SimpleDateFormat DateFormat = new SimpleDateFormat("MM/dd/yyyy hh:mm:ss aa");
```

Comment:
Move to an instance variable since creating SimpleDateFormats is very expensive.